tf_encrypted/convert/convert.py

```diff
@@ -83,7 +83,7 @@ def convert(
       if node.name not in specop_outputs:
 
         output = strip_tensor_info(node.name)
-        inputs = [strip_tensor_info(x) for x in node.input]
+        inputs = [x for x in node.input]
```

Comment:
why this change?

Reply:
ah it's because `strip_tensor_info` removes everything after `:` in the node name. However, at least for `split` operation, the outputs are distinguished by what's after `:`. For example, for split (in 3), it will be `split`, `split:1`, `split:2`. If we remove everything after `:`, the first output will be re-used three times.

Reply:
awesome, thanks for clarifying!